src/ui/server/controllers/insurance/policy/map-and-save/loss-payee-financial-details-uk/index.ts

```diff
@@ -0,0 +1,45 @@
+import hasFormData from '../../../../../helpers/has-form-data';
+import mapSubmittedData from '../../map-submitted-data/loss-payee-financial-details-uk';
+import save from '../../save-data/loss-payee-financial-details-uk';
+import { Application, RequestBody, ValidationErrors } from '../../../../../../types';
+
+/**
+ * lossPayeeFinancialDetailsUk
+ * mapAndSave lossPayeeFinancialDetails
+ * Map and save any valid nominatedLossPayee fields
+ * @param {Express.Request.body} Express request body
+ * @param {Application}
+ * @param {Object} Validation errors
+ * @returns {Boolean}
+ */
+const lossPayeeFinancialDetailsUk = async (formBody: RequestBody, application: Application, validationErrors?: ValidationErrors) => {
+  try {
+    if (hasFormData(formBody)) {
+      const populatedData = mapSubmittedData(formBody);
+
+      let saveResponse;
+
+      if (validationErrors) {
+        saveResponse = await save.lossPayeeFinancialDetailsUk(application, populatedData, validationErrors.errorList);
+      } else {
+        saveResponse = await save.lossPayeeFinancialDetailsUk(application, populatedData);
+      }
+
+      if (!saveResponse) {
```

Comment:
Same as above suggestion to keep it simple please.

Reply:
No need to do this IMO, there is already logging and/or errors thrown in the `save` function and the GQL call itself, so these would be logged.

If we add logging here I think it will just be adding noise.